homeassistant/components/husqvarna_automower/lawn_mower.py

```diff
@@ -98,12 +99,21 @@ def activity(self) -> LawnMowerActivity:
 
     async def async_start_mowing(self) -> None:
         """Resume schedule."""
-        await self.coordinator.api.resume_schedule(self.mower_id)
+        try:
+            await self.coordinator.api.resume_schedule(self.mower_id)
+        except ApiException as exception:
+            _LOGGER.error("Command couldn't be sent to the command que: %s", exception)
```

Comment:
Please raise `HomeAssistantError` with the message as exception argument instead of logging the message.

The message will show in the frontend.

https://developers.home-assistant.io/docs/integration_quality_scale_index#silver-